tests/unit/models/category-test.js

```diff
@@ -14,8 +18,9 @@ test('it exists', function(assert) {
 });
```

Comment:
Thoughts on just deleting this boilerplate test?  We essentially test it exists in the test below too.